pkg/util/ipset/ipset.go

```diff
@@ -294,22 +369,22 @@ func (runner *runner) CreateSet(set *IPSet, ignoreExistErr bool) error {
 // If ignoreExistErr is set to true, then the -exist option of ipset will be specified, ipset ignores the error
 // otherwise raised when the same set (setname and create parameters are identical) already exists.
 func (runner *runner) createSet(set *IPSet, ignoreExistErr bool) error {
-	args := []string{"create", set.Name, string(set.SetType)}
-	if set.SetType == HashIPPortIP || set.SetType == HashIPPort || set.SetType == HashIPPortNet || set.SetType == HashIP {
+	args := []string{"create", set.Name(), string(set.SetType())}
+	if set.SetType() == HashIPPortIP || set.SetType() == HashIPPort || set.SetType() == HashIPPortNet || set.SetType() == HashIP {
```

Comment:
This is in the same package - it can just use fields rather than methods.  Does that make it possible to remove more methods?